src/library_egl.js

```diff
@@ -281,17 +281,24 @@ var LibraryEGL = {
       }
       contextAttribs += 8;
     }
-    if (glesContextVersion != 2) {
+    if (glesContextVersion != 2 && glesContextVersion != 3) {
 #if GL_ASSERTIONS
-      Module.printErr('When initializing GLES2/WebGL1 via EGL, one must pass EGL_CONTEXT_CLIENT_VERSION = 2 to GL context attributes! GLES version ' + glesContextVersion + ' is not supported!');
+      Module.printErr('When initializing GLES2/WebGL1 via EGL, one must pass EGL_CONTEXT_CLIENT_VERSION = 2 or 3 to GL context attributes! GLES version ' + glesContextVersion + ' is not supported!');
```

Comment:
This says "When initializing GLES2/WebGL1 via EGL, one must pass EGL_CONTEXT_CLIENT_VERSION = 2 or 3 to GL context attributes!". Perhaps better to have "When initializing GLES2/3 or WebGL1/2 via EGL, one must pass EGL_CONTEXT_CLIENT_VERSION = 2 or 3 to GL context attributes!"?